streaming/consumer.cc

```diff
@@ -35,6 +64,9 @@ std::function<future<> (flat_mutation_reader_v2)> make_streaming_consumer(sstrin
             //FIXME: for better estimations this should be transmitted from remote
             auto metadata = mutation_source_metadata{};
             auto& cs = cf->get_compaction_strategy();
+
+            offstrategy = maybe_disable_offstrategy(db.local(), cs, cf->schema(), offstrategy);
+
             // Data segregation is postponed to happen during off-strategy if latter is enabled, which
             // means partition estimation shouldn't be adjusted.
             const auto adjusted_estimated_partitions = (offstrategy) ? estimated_partitions : cs.adjust_partition_estimate(metadata, estimated_partitions, cf->schema());
```

Comment:
Will this trigger the original "auto increase min_threshold" logic for streamed data?

Reply:
Will the streamed data be segregated when off-strategy is disabled?

Reply:
I think I got it too. I see we have our code nicely instrumented to be able to disable off-strategy - nice planning!